src/store/Store.ts

```diff
@@ -59,18 +59,23 @@ export interface GraphState {
   } | null;
 }
 
-export interface Token {
+export enum LoginStatus {
+  logging,
+  loggedIn,
+  loggedOut,
+  error
+}
+
+export interface Session {
   token: string;
-  expired_at: string;
+  expiresOn: string;
```

Comment:
I assume `expiresOn` is some kind of _date_ (or moment now) type that the component can format instead of passing an already formatted string value.

Reply:
It actually is the date response from the server itself. I thought about making it a more high level definition with Moment, but that would require some kind of preprocessing that to be honest I'm not much of a fan.